### PR TITLE
on click value to bug

### PR DESCRIPTION
**Pull request by Mattchewone**
Fixes https://github.com/canjs/can-stache-bindings/issues/484

### CODE REVIEW

File: test/colon/event-test.js [hunk 1]
```diff
@@ -591,4 +591,21 @@ testHelpers.makeTests("can-stache-bindings - colon - event", function(name, doc,
 
 		QUnit.equal(map.get("clickCount"), 1, "function got called");
 	});
+
+	testIfRealDocument("on:click:value:to on button (#484)", function() {
```
Comment (chasenlehara) [inline test/colon/event-test.js:595]: I read your [commit message](https://github.com/canjs/can-stache-bindings/pull/507/commits/b97ede71c78110c72d9d92df1041920c8fedf365) but I don’t quite understand, why does this test use `testIfRealDocument`?

Comment (Mattchewone) [reply test/colon/event-test.js:595]: The `button` element in `vdom` doesn't contain `value` property. I spoke with @phillipskevin and @matthewp and they suggested to not run the test when in `vdom`.

Comment (matthewp) [reply test/colon/event-test.js:595]: Did you figure out what the purpose of the value property on a button is?

**Review by phillipskevin — APPROVED** [review]

File: test/colon/event-test.js (hunk 1 above)
Comment (Mattchewone) [reply test/colon/event-test.js:595]: @matthewp According to Mozilla docs here - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

Value:
> The initial value of the button. It defines the value associated with the button which is submitted with the form data. This value is passed to the server in params when the form is submitted.

**Review by justinbmeyer — APPROVED** [review]

File: test/colon/event-test.js (hunk 1 above)
Comment (matthewp) [reply test/colon/event-test.js:595]: Interesting, so it sounds a bit like `<input type=hidden>` then, but on a button. Weird.

**Pull request merged**